package.json

```diff
@@ -84,13 +84,18 @@
     "prism-languages": "^0.3.3",
     "prismjs": "^1.9.0",
     "raw-loader": "^0.5.1",
+    "remark-autolink-headings": "^5.0.0",
+    "remark-loader": "^0.3.0",
+    "remark-mermaid": "^0.2.0",
     "request": "^2.81.0",
     "sass-loader": "^6.0.6",
     "sitemap-static": "^0.4.2",
+    "static-site-generator-webpack-plugin": "^3.4.1",
     "style-loader": "^0.18.2",
     "tap-min": "^1.2.1",
     "tap-parser": "^6.0.1",
     "through2": "^2.0.3",
+    "uglifyjs-webpack-plugin": "^1.1.6",
     "url-loader": "^0.5.9",
     "webpack": "^3.10.0",
```

Comment:
Can we also update to use webpack 4? :grin:

Reply:
@dhruvdutt yeah but it was decided to do as a follow up right after we merge this into master to not hold it anymore. I will just apply part of my WP4 changes from master into newer `master` after this merged so its on webpack 4 and also update the configs according to new build setup

Reply:
Sounds like a plan. :100: